daemon/algod/api/client/restClient.go

```diff
@@ -221,14 +221,20 @@ func (client RestClient) TransactionsByAddr(addr string, first, last, max uint64
 	return
 }
 
+// AssetInformation gets the AssetInformationResponse associated with the passed asset index
+func (client RestClient) AssetInformation(index uint64) (response v1.AssetParams, err error) {
+	err = client.get(&response, fmt.Sprintf("/assets/%d", index), nil)
+	return
+}
+
 // AccountInformation also gets the AccountInformationResponse associated with the passed address
 func (client RestClient) AccountInformation(address string) (response v1.Account, err error) {
 	err = client.get(&response, fmt.Sprintf("/account/%s", address), nil)
 	return
 }
 
-// AssetInformation gets the AssetInformationResponse associated with the passed asset creator and index
-func (client RestClient) AssetInformation(creator string, index uint64) (response v1.AssetParams, err error) {
+// AssetInformationWithCreator gets the AssetInformationResponse associated with the passed asset creator and index
```

Comment:
What's the point of this API if `AssetInformation()` exists?

Reply:
I didn't want it either, but was asked to leave it -- I just asked rotem again though and now we can remove it :upside_down_face: